tests/conditional_compilation/conftest.py

```diff
@@ -7,9 +7,10 @@
 """ Pytest configuration for compilation tests.
 
 Sample usage:
-python3 -m pytest --artifacts ./compiled --test_conf=<path to test config> \
-    --sea_runtool=./IntelSEAPI/runtool/sea_runtool.py \
-    --benchmark_app=./bin/benchmark_app test_collect.py
+python3 -m pytest --test_conf=<path to test config> \
```

Comment:
We don't need to clone `IntelSEA` original repo. Please use our version: https://github.com/openvinotoolkit/openvino/tree/master/thirdparty/itt_collector